ups/eupspkg.cfg.sh

```diff
@@ -6,37 +6,26 @@ config()
     ARGS=()
 
     ARGS+=("-DCMAKE_INSTALL_PREFIX=${PREFIX}")
+
+    # Remove mysql-test directory (~330 MB) to reduce installation size
+    ARGS+=("-DINSTALL_MYSQLTESTDIR=")
+
     # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
     ARGS+=('-DPLUGIN_TOKUDB=NO')
 
     # Due to cmake library discovery being overly energetic (searching every
     # lib directory relative to each entry in $PATH) we use the bundled SSL and
     # ZLIB libraries to avoid link confusion downstream
-
-    ARGS+=('-DWITH_SSL=bundled')
     ARGS+=('-DWITH_ZLIB=bundled')
     ARGS+=('-DWITH_PCRE=bundled')
 
-    case $(uname) in
-        Linux*)
-            if grep -q -i "CentOS release 5" /etc/redhat-release; then
-                # mroonga does not build on EL5 (conda ref platform) and is
-                # unused by LSST
-                ARGS+=('-DPLUGIN_MROONGA=NO')
-                # required to build on EL5
-                ARGS+=('-DHAVE_CXX_NEW=NO')
-            fi
-            ;;
-        Darwin*)
-            # No special arguments needed (must use bundled SSL)
-
-            ;;
-        *)
-            # non-fatal
-            echo "unsupported platform: $(uname)"
-            ;;
-    esac
+    # Prevent CMake from finding and linking against libraries distributed in $(dirname
+    # python)/../lib. This is CMake's default behavior, but can cause us to erroneously link
+    # against libraries distributed by e.g. Anaconda.
+    PYTHONLIBDIR=$(which python | sed -e's|bin/python|lib|')
+    ARGS+=("-DCMAKE_SYSTEM_IGNORE_PATH=${PYTHONLIBDIR}")
```

Comment:
For the same reason it was already in `mariadb`. If cmake finds a library in the Anaconda lib then all the builds fail because nothing else is looking in that library directory. This lets us mask out Anaconda from the search path.

Reply:
Can you give me an example of that happening?  I've built the entire stack with this removed.

Reply:
All of the examples that caused us to add it in the first place. The problem is that *sometimes* someone puts a library in their conda installation that we aren't expecting and then everything breaks. In your case you probably hadn't installed any extra packages. DM-5595 was the relevant Jira ticket. In that case it was `libz` (which we now bypass by specifically asking for the internal one) but in general we don't want cmake to ever find any library in the conda lib path.